src/font.cpp

```diff
@@ -50,7 +50,7 @@ namespace {
 		if(ret != (data + size) && ret->code == code) {
 			return ret;
 		} else {
-			static ShinonomeGlyph const empty_glyph = { 0, true, {0} };
+			static ShinonomeGlyph const empty_glyph = { 65533, true, { 96, 240, 504, 924, 1902, 3967, 4031, 1982, 1020, 440, 240, 96 } };
```

Comment:
Small suggestion: Maybe rename `empty_glyph` to `replacement_glyph`, because it is not empty anymore.